packages/mobile/scripts/run_app.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# ====================================
+# Configure and run the mobile app
+# ====================================
+
+# Flags:
+# -n: Name of the network to run on
+# -p: Platform (android or ios)
+# -f: Fast (skip steps not required unless network or depedencies changes)
+# -h: Hot Reload (Restore nav state on reload)
```

Comment:
`-h` is usually used for help, your suggestion of `-r` is better IMHO.
Also would rename `Hot Reload` to what it does: "Restore nav state on reload"

Reply:
Will do!